org.jellyfin.webos/js/index.js

```diff
@@ -272,18 +272,81 @@ function abort() {
     console.log("Aborting...");
 }
 
+function injectScript(document, url) {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url);
+    xhr.onload = function () {
```

Comment:
shouldn't we at least `alert()` a user if xhr failed?

Reply:
`alert` is not displayed. I can probably try to preload the injected script before actually connecting to the server, and then `displayError` in case of xhr failure.